GPDemo/src/main/resources/application.properties

```diff
@@ -0,0 +1,18 @@
+spring.application.name=GPDemo
+
+server.port=8094
+
+spring.jpa.hibernate.ddl-auto=create
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+
+spring.datasource.url=jdbc:mysql://localhost:3306/fruitsDB?createDatabaseIfNotExist=true
+spring.datasource.username=root
```

Comment:
Its not recommended to connect to DB using the root user, instead create a new user to connect with